homeassistant/components/frontend/__init__.py

```diff
@@ -291,3 +351,21 @@ def get(self, request):    # pylint: disable=no-self-use
         """Return the manifest.json."""
         msg = json.dumps(MANIFEST_JSON, sort_keys=True).encode('UTF-8')
         return web.Response(body=msg, content_type="application/manifest+json")
+
+
+class ThemesView(HomeAssistantView):
+    """View to return defined themes."""
+
+    requires_auth = False
+    url = '/api/themes'
+    name = 'api:themes'
+
+    @callback
+    def get(self, request):
+        """Return themes."""
+        hass = request.app['hass']
+
+        return self.json({
+            'themes': hass.data[DATA_THEMES],
+            'current_theme': hass.data[DATA_CURRENT_THEME],
```

Comment:
let's rename this to `default_theme` (so it can be overridable per browser in the future).